spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/DefaultPubSubAcknowledger.java

```diff
@@ -27,23 +27,35 @@
  * ack IDs and subscription name.
  *
  * @author João André Martins
+ * @author Doug Hoard
  */
 class DefaultPubSubAcknowledger implements PubSubAcknowledger {
 
 	private SubscriberStub subscriberStub;
 
+	private boolean asyncAck;
+
 	DefaultPubSubAcknowledger(SubscriberStub subscriberStub) {
 		this.subscriberStub = subscriberStub;
 	}
 
+	public void setAsyncAck(boolean asyncAck) {
```

Comment:
Why do you still insist in adding `Ack` suffix to the obvious option?
The `DefaultPubSubAcknowledger` does nothing more unless `(n)ack`.
So, what is the point to be so verbose?
Plus where is a JavaDoc on this new public API?

Reply:
Removed the verbosity since the method is implicit for the classes function.